media-video/ffmpeg/files/amf-env-vulkan-override

```diff
@@ -0,0 +1 @@
+VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/amd_pro_icd64.json
```

Comment:
Will this work with other arches beside amd64?

Reply:
So basically, `ffmpeg` can be built with `amf` support in any arch a priori. But as of now it needs a library that is only offered by the `amd64` `amdgpu-pro-amf` package. So the `amf` useflag is masked in every arch except `amd64`. 

A little subtlety that I am not sure about is that `ffmpeg` can be built in multilib, abi 32 and 64, with `amf` support. But I don't know if the `amf` code path would work in the 32 bit abi. But I need to enable building ffmpeg with amf support in abi 32 otherwise, to have a multilib amf, one would need to disable the `amf` flag.